Gordon360/Models/ViewModels/StudentNewsImageUploadViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Gordon360.Models.ViewModels
```

Comment:
If your view model is only ever going to have one field, I would recommend just using the native type of that field instead of having a view model for it. i.e. delete this view model and pass a `string` into `EditPostingImage(...)` & other methods.